### DISCUSSION TITLE
Tesla Powerwall MLR

### DISCUSSION THREAD

**Discussion by rob0101** [amberelectric/public-api#205] [General]
Is there a reason why Smartshift does not read the MRL from the Tesla API instead of asking the customer to manually sync it via the app ?

Lots of us are now using Netzero and other tools to schedule adjustments to the MRL.    I have 3 different MRLs per day.

Smartshift may want/need to charge a battery if it thinks the MRL is higher than it actually is, so things get messy.

**Comment by madpilot** [comment]
It’s not available via the API we use.

**Reply by rob0101** [reply]
> It’s not available via the API we use.

Wow. Tesla do some weird stuff.
